kickstart/provision_atomic.erb

```diff
@@ -18,7 +18,9 @@ bootloader --timeout=3
 text
 
 <% if @host.os.name.match /.*fedora.*/i -%>
-ostreesetup --nogpg --osname=fedora-atomic --remote=fedora-atomic --url=<%= @host.param_true?('atomic-upstream') ? "https://dl.fedoraproject.org/pub/fedora/linux/atomic/#{@host.os.major}/" : "#{@host.operatingsystem.medium_uri(@host)}/content/repo/" %> --ref=fedora-atomic/f<%= @host.os.major %>/<%= @host.architecture %>/docker-host
+# Use @host.operatingsystem.medium_uri(@host)}/content/repo/ as the URL if you
+# have set up a local installation media for Fedora
+ostreesetup --nogpg --osname=fedora-atomic --remote=fedora-atomic --url=https://dl.fedoraproject.org/pub/fedora/linux/atomic/#{@host.os.major}/ --ref=fedora-atomic/f<%= @host.os.major %>/<%= @host.architecture %>/docker-host
```

Comment:
My bad - full context here https://github.com/theforeman/foreman/pull/3111 . In short, the medium URL where the PXE stuff is fetched from, is different from the URL where the kickstart trees for `ostreesetup` are fetched from.